utils/sv_easing_functions.py

```diff
@@ -328,35 +328,46 @@ def BounceEaseInOut(p, settings=defaultBounceSettings):
 
 
 easing_dict = {
+    # LINEAR
```

Comment:
(as an academic discussion) It only seem care about the keys to access the name to add it to a menu list. Correct? (i checked this code before i made the changes btw).  So due to key reordering, this menu items will be reordered too, but that doesn’t really change the behavior of the node.  For this node in particular finding a name in a 30 item list is kind of hard and I always have to scan the whole list to find what i’m looking for (which is why I though reordering of this list is not a big deal). Also the drop down can reverse the order depending on where on the screen the node (drop up vs drop down) so even if you’re used to knowing where an item is in the list, the reversing makes it tricky to find it again.